grpc-native/src/main/java/org/ballerinalang/net/grpc/builder/syntaxtree/SyntaxTreeGenerator.java

```diff
@@ -105,7 +107,18 @@ public static SyntaxTree generateSyntaxTree(StubFile stubFile) {
         NodeList<ImportDeclarationNode> imports = NodeFactory.createEmptyNodeList();
         if (stubFile.getStubList().size() > 0) {
             ImportDeclarationNode importForGrpc = Imports.getImportDeclarationNode("ballerina", "grpc");
-            imports = AbstractNodeFactory.createNodeList(importForGrpc);
+            ImportDeclarationNode importForTime = null;
+            for (Object desc : stubFile.getDescriptors().toArray()) {
```

Comment:
Shall we move this logic to a utils function that scans a `stubFile` object and returns a `boolean`. If the function returns `true`, add `import ballerina/time` here?
That way it would be easier to maintain if we decide to support another type in the future.

Reply:
fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/312/commits/94b649a0e4f37a25673b580f6a7fde9154645a04